static/js/millennium_diff_table.js

```diff
@@ -235,38 +235,29 @@ function matchResources(conformanceData, capabilityStatementData) {
 }
 
 /**
- * Merges the two sorted arrays alphabetically.
+ * Merges the two sorted arrays alphabetically. Entries are ordered by their R4 name first, if no R4 resource
+ * exists then the DSTU2 is used.
  * @param {array} includesR4 - The array of resource objects that contain R4 resources.
  * @param {array} dstu2Only - The array of resource objects that only contain DSTU2 resources.
  * @returns {array} The resultant array after merging.
  */
 function mergeSortedArrays(includesR4, dstu2Only) {
   let result = [];
-  let i = 0;
-  let j = 0;
```

Comment:
nit: Defining the lets outside the loop like it was before is a JavaScript standard.

Reply:
`let`'s are block scoped. Meaning by definition, if we defined them at the top of the function, they will be available outside of the loops (where they are only needed). Given that, previously, it was a `where` loop, versus now a `for` loop. And I can't find anything in eslint regarding this or anything stating this for a `for` loop in general, this seems unnecessary (and potentially introduce more bugs).

Reply:
_All_ variables in JavaScript are defined at the top of the method. That's why my personal preference for any JavaScript is to define all variables at the top to remind readers of that.

The new ES6 syntax for `let` just avoids hoisting.